aiida_mlip/calculations/geomopt.py

```diff
@@ -117,16 +112,28 @@ def prepare_for_submission(
         calcinfo = super().prepare_for_submission(folder)
         codeinfo = calcinfo.codes_info[0]
 
-        opt_kwargs = self.inputs.opt_kwargs.get_dict()
-
-        geom_opt_cmdline = {
-            "traj": self.inputs.traj.value,
-            "fully-opt": self.inputs.fully_opt.value,
-            "vectors-only": self.inputs.vectors_only.value,
-            "fmax": self.inputs.fmax.value,
-            "steps": self.inputs.steps.value,
-            "opt-kwargs": opt_kwargs,
-        }
+        geom_opt_cmdline = {"traj": self.inputs.traj.value}
+        if "opt_kwargs" in self.inputs:
```

Comment:
What happens if they aren't? Are defaults handled in the called routines?

Reply:
if they aren't in inputs they might be in the config, if they are not in the config either it will use whatever default janus has for them.
I used to have the default in aiida too (that anyways it matched janus one) but I removed it because otherwise it would use the default and override the config file. We only want that when we intentionally set an input